app/src/main/java/com/example/androidtrainingproject/ui/home/HomeScreen.kt

```diff
@@ -162,6 +229,136 @@ fun Home(navController: NavController) {
                     }
                 }
             }
+
+            if (sheetState.isVisible) {
+                ModalBottomSheet(
+                    modifier = Modifier.fillMaxSize(),
+                    onDismissRequest = {
+                        coroutineScope.launch {
+                            sheetState.hide()
+                        }
+                    },
+                    sheetState = sheetState
+                ) {
+                    Column(
+                        modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.padding_l))
```

Comment:
I talked about exposing modifier for outer paddings in the extracted composables below but think about adding a 

```
verticalArrangement = Arrangement.spacedBy(
  dimensionResource(id = R.dimen.padding_m)
)
```

on the whole `Column` as that seems to be the common padding between all of the filter groups anyway. Better to have the padding at one place instead of on two/three of them.